src/applications/personalization/dashboard/components/debts/Debts.jsx

```diff
@@ -128,11 +135,13 @@ const BenefitPaymentsAndDebt = ({
             </DashboardWidgetWrapper>
           </>
         )}
-        {debtsCount > 0 && (

```

Comment:
Let's bring back this count check, don't want to lose out on existing functionality 🤙

Reply:
Righto